### DISCUSSION TITLE
A transaction never arrive to my wallet

### DISCUSSION THREAD

**Discussion by panchimilano** [Chia-Network/chia-blockchain#6663] [Support]
Few days ago a small amount of chia has been send to my wallet the amount never arrive . what I can do 
this is the transaction .
https://www.chiaexplorer.com/blockchain/coin/0xdd3bda0dfe1e4d5853a263bec419f6ea8f412e9a84c3d5b92c8e344305b2e4fe
thanks in advance for your help .

**Comment by loppefaaret** [comment]
make sure you wallet is synced passed the block the transaction was committed to the blockchain in.
https://www.chiaexplorer.com/blockchain/address/xch1z2j3ml9n7xsmn8qa7jlst7z3cy942l8sev7ep8p659ax5qm7ckwswg70a8 shows it there

**Comment by BrandtH22** [comment]
@panchimilano , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
